exercises/F-strings-methods/exercise.js

```diff
@@ -6,8 +6,13 @@
 // var myGreeting = 'My name is ' + myName + ' and my name is ' + myName.length + ' characters long';
 
 function getGreeting(name) {
-    var myGreeting = 'My name is ' + name + ' and my name is ' + name.length + ' characters long';
-    return myGreeting;
+  var myGreeting =
+    "My name is " +
+    name +
+    " and my name is " +
+    name.length +
+    " characters long";
+  return myGreeting;
```

Comment:
Great Job! As a challenge, you can make use of template literals to tidy up the code and make it easier to read. This link: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals and video: https://youtu.be/NgF9-pdTDGs are quite useful - should you need them.

Reply:
Thank you! It auto-formatted that way on save :-)